loops.js

```diff
@@ -6,8 +6,16 @@
 // Hoera!
 // ==========================================
 
+console.log(" ================= Opdracht 1 =========================")
 
 
+const juichen = "Hoera!";
+
```

Comment:
Conventie is engelse termen gebruiken ipv Nederlandse woorden

Reply:
Je hebt helemaal gelijk. Zal ik aanpassen en voortaan consequent doorvoeren.